instaclustr/structs.go

```diff
@@ -176,6 +183,19 @@ type VPCPeering struct {
 	PeerRegion         string        `json:"peerRegion"`
 }
 
+type GCPVPCPeering struct {
+	ID                 string `json:"id"`
+	Name               string `json:"name"`
+	GCPVpcConnectionID string `json:"peerProjectId"`
```

Comment:
**[Question]** I'm guessing `peerProjectId` is GCP's naming convention, but do we use that internally in the API orr do we use `GCPVpcConnectionID`? Would be good to keep them consistent

Reply:
ok have changed that to `PeerProjectID` keeping it consistent with camel casing